apps/admin-gui/src/app/shared/components/dialogs/sponsor-this-member-dialog/sponsor-this-member-dialog.components.html

```diff
@@ -0,0 +1,35 @@
+<div class="{{theme}}">
+  <h1 mat-dialog-title>{{'DIALOGS.SPONSOR_THIS_MEMBER.TITLE' | translate}}</h1>
+  <div mat-dialog-content>
+    <app-choose-sponsor
```

Comment:
Please make this component's 'Choose sponsor for created user' header more generic. We are not creating a new user in this context (it makes sense only in the dialog for creating a new sponsored member, for that matter)
![Screenshot from 2023-07-10 12-39-53](https://github.com/CESNET/perun-web-apps/assets/97669007/cd124572-edeb-49eb-b0d6-fc1a0cf21cee)